qstrader/trading_session/backtest.py

```diff
@@ -10,29 +18,81 @@ class Backtest(object):
     carrying out an event-driven backtest.
     """
     def __init__(
-        self, price_handler,
-        strategy, portfolio_handler,
-        execution_handler,
-        position_sizer, risk_manager,
-        statistics, equity,
-        sentiment_handler=None
+        self, config, strategy, tickers,
+        equity, start_date, end_date, events_queue,
+        price_handler=None, portfolio_handler=None,
+        compliance=None, position_sizer=None,
+        execution_handler=None, risk_manager=None,
+        statistics=None, sentiment_handler=None,
+        title=None, benchmark=None
     ):
         """
         Set up the backtest variables according to
         what has been passed in.
         """
-        self.price_handler = price_handler
+        self.config = config
         self.strategy = strategy
```

Comment:
I presume you mean multiple derived Strategy class objects for the same Backtest? If so, I don't think so. If you have an idea of the API that you'd like to see for that, I could include it in this branch and then include an example set of strategies to test it.

Let me know ;-)